Add contents and optimized the list

mingcheng opened the pull request:
Hello, I am [mingcheng](https://github.com/mingcheng) from the [Ant Group Open Source Office](https://github.com/antgroup). I am responsible for the technical evangelism of open-source projects.

I thought your project was awesome, and it was very helpful to us. Simultaneously, we hope to obtain the latest status of the project synchronously (for example, the links of many projects may not be available), so we improved the presentation form of the list.We used emoji to mark the status and popularity of each project. At the same time, we also provide corresponding scripts to check the availability of each project status so that we can update the list regularly.

Thank you for providing this nice project. We hope to improve the open-source community together. Please keep in touch at any time.

mingcheng committed:
docs(README): update contents and new sections 📚

- Revamp table of contents with emoji icons for visual clarity
- Add new sections for Governance, Policy, Community, and OSPO Resources
- Update existing entries with fire emojis for popular tools
- Restructure categories for better navigation and readability
- Ensure consistency in formatting across all sections

Signed-off-by: mingcheng <mingcheng@apache.org>

justaugustus commented on scripts/README.md:
Please remove.
(Same comment as previous files.)

justaugustus commented on README.md:
```diff
@@ -1,137 +1,211 @@
-
-## Code Reviews
+This list identifies packages and projects that have been built by [TODO Group
+members](https://github.com/todogroup/awesome-ospo) and [Ant Group OSPO](https://github.com/antgroup/) or found helpful for managing open source projects and offices.
```
Please remove the mention of Ant Group here.
While we have several contributors to this repo, we do not currently list them individually (and so it would be inappropriate to list just one in this PR).

justaugustus reviewed (changes requested):
Hey @mingcheng — thanks for the great draft contribution!

When you have a chance, please:

- rebase the PR to resolve the merge conflicts
- address the initial feedback I've provided
- try your best to update the PR description to match the [updated PR template](https://github.com/todogroup/awesome-ospo/blob/main/.github/pull_request_template.md) that we've [recently merged](https://github.com/todogroup/awesome-ospo/pull/74)
- re-request a review from me once you've had an opportunity to do the previous steps

mingcheng committed:
docs(README): fix descriptions for consistency 📚

- Add trailing periods to tool descriptions in Code Review, CI/CD, and GitHub Management sections
- Capitalize first word in pull-review and Gitleaks descriptions
- Prefix "The" to TODO Group description in OSPO Resources section
- Adjust sentinel description to lowercase "test" for uniformity

Signed-off-by: mingcheng <mingcheng@apache.org>
